web/modules/custom/dpl_react_apps/src/Controller/DplReactAppsController.php

```diff
@@ -345,6 +347,20 @@ public static function dplCmsBaseUrl(): string {
     return rtrim($url, "/");
   }
 
+  /**
+   * Get the base url of the API's exposed by this site.
+   *
+   *  An array of base urls.
```

Comment:
This looks like leftovers from a missing `@return` statement :smile:

Reply:
Fixed